pkgs/top-level/cuda-packages.nix

```diff
@@ -81,6 +83,45 @@ let
     nccl = final.callPackage ../development/cuda-modules/nccl { };
     nccl-tests = final.callPackage ../development/cuda-modules/nccl-tests { };
 
+    tests =
+      let
+        bools = [
+          true
+          false
+        ];
+        configs = attrsets.genAttrs [
+          "openCVFirst"
+          "useOpenCVDefaultCuda"
+          "useTorchDefaultCuda"
+        ] (trivial.const bools);
```

Comment:
I'm sorry but I don't see how this is easier to read or write than `{ openCVFirst = bools; ... }` xD

Reply:
Yeah, I got carried away refactoring -- I'll change that part back :)

Reply:
Resolved in latest